tests/test_schema_text.py

```diff
@@ -0,0 +1,50 @@
+import os
+from json import load
+
+from structlog import configure
+from structlog import getLogger
+from structlog.stdlib import LoggerFactory
+
+from app.validation.validate_schema_text import validate_json_schema_text
+
+logger = getLogger()
+
+configure(logger_factory=LoggerFactory())
+
+
+def validate_schema_text(json_to_validate):
+    errors = validate_json_schema_text(json_to_validate)
+    return errors
+
+
+def _open_and_load_schema_file(file):
+    json_file = open(os.path.join(os.path.dirname(os.path.realpath(__file__)), file), encoding='utf8')
```

Comment:
use a context manager to open a file

Reply:
Amended the open and load function to use a context manager